test/testHeadless.m

```diff
@@ -18,9 +18,12 @@
 % The width and height are specified to circumvent different DPIs in developer
 % machines. The float format reduces the probability that numerical differences
 % in the order of numerical precision disrupt the output.
-    extraOptions = {'width' ,'\figurewidth' ,...
-                    'height','\figureheight',...
-                    'floatFormat', '%8.6g'  ,...
+    extraOptions = {'width' ,'\figureWidth', ...
+                    'height','\figureHeight',...
+                    'floatFormat', '%8.3f',  ...
```

Comment:
I saw that beauty but (pdf|lua)LaTex compile happily. This is what a number in the range of 10^45 looks like ;-)